docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_position: 1
+description: An introductory modding guide that will show you how to swap 47's Signature Suit with an NPC.
+---
+
+# Recommended reading 
+* [Glacier 2 Modding Basics](https://wiki.notex.app/glacier2/modding_basics)
+* [Chunk Data](https://wiki.notex.app/glacier2/chunkdata)

```

Comment:
As in `[Glacier 2 Modding Basics](/glacier2/modding_basics)`? Because I checked through a few other documents, and saw that modding_basics.md uses direct links as well, e.g. `[here](https://wiki.notex.app/glacier2/fileformats/).`